stdlib/xml/dom/minidom.pyi

```diff
@@ -87,31 +90,31 @@ class NamedNodeMap:
     def keys(self): ...
     def keysNS(self): ...
     def values(self): ...
-    def get(self, name, value: Any | None = ...): ...
+    def get(self, name: str, value: Any | None = ...): ...
     def __len__(self) -> int: ...
     def __eq__(self, other: object) -> bool: ...
-    def __ge__(self, other: Any) -> bool: ...
-    def __gt__(self, other: Any) -> bool: ...
-    def __le__(self, other: Any) -> bool: ...
-    def __lt__(self, other: Any) -> bool: ...
+    def __ge__(self, other: object) -> bool: ...
+    def __gt__(self, other: object) -> bool: ...
+    def __le__(self, other: object) -> bool: ...
+    def __lt__(self, other: object) -> bool: ...
     def __getitem__(self, attname_or_tuple): ...
-    def __setitem__(self, attname, value) -> None: ...
-    def getNamedItem(self, name): ...
-    def getNamedItemNS(self, namespaceURI: str, localName): ...
-    def removeNamedItem(self, name): ...
-    def removeNamedItemNS(self, namespaceURI: str, localName): ...
-    def setNamedItem(self, node): ...
-    def setNamedItemNS(self, node): ...
-    def __delitem__(self, attname_or_tuple) -> None: ...
+    def __setitem__(self, attname: str, value: Attr | str) -> None: ...
+    def getNamedItem(self, name: str) -> Attr | None: ...
+    def getNamedItemNS(self, namespaceURI: str, localName: str | None) -> Attr | None: ...
+    def removeNamedItem(self, name: str) -> Attr: ...
+    def removeNamedItemNS(self, namespaceURI: str, localName: str | None): ...
+    def setNamedItem(self, node: Attr) -> Attr: ...
+    def setNamedItemNS(self, node: Attr) -> Attr: ...
+    def __delitem__(self, attname_or_tuple: str) -> None: ...
```

Comment:
This can also be a tuple, more precisely a `tuple[str, str | None]`. `__getitem__` takes the same type.

Reply:
Sorry about that, I had mixed up which line that was accurate for as one location they're used as arguments to `_attrs` I believe and the other they're used as method arguments